### PR TITLE
makefile release process

### PR DESCRIPTION
**Pull request by gijsio**
makefile combines libraries for shields into separate folders

Im not sure if this is the correct makefile procedure, but though to get started on it anyways

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+pysense:
+	mkdir pysense
+	mkdir pysense/lib
+	# sensors
+	cp shields/lib/LIS2HH12.py pysense/lib/
+	cp shields/lib/LTR329ALS01.py pysense/lib/
+	cp shields/lib/MPL3115A2.py pysense/lib/
+	cp shields/lib/SI7006A20.py pysense/lib/
+	# pycoproc
+	cp shields/lib/pycoproc_1.py pysense/lib/pycoproc.py
```
Comment (peter-pycom) [inline Makefile:14]: the problem, when we rename it is, that we also need to adapt the imports ... which we could do with sed, but ugh ... more and more magic in this 'release' process that needs maintaining. I'm not sure either way....

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	# example
+	cp shields/pysense_1.py pysense/main.py
+	zip -r pysense.zip pysense
```
Comment (peter-pycom) [inline Makefile:17]: I would prefer if each rule cleans up it's temp folder

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	cp shields/lib/MPL3115A2.py pysense2/lib/
+	cp shields/lib/SI7006A20.py pysense2/lib/
+	# pycoproc
+	cp shields/lib/pycoproc.py pysense/lib/pycoproc.py
+	# example
+	cp shields/pysense_2.py pysense/main.py
+	zip -r pysense2.zip pysense2
+
+clean:
+	@echo "Cleaning up files"
```
Comment (peter-pycom) [inline Makefile:34]: zip files here

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	mkdir pysense2/lib
+	# sensors
+	cp shields/lib/LIS2HH12.py pysense2/lib/
+	cp shields/lib/LTR329ALS01.py pysense2/lib/
+	cp shields/lib/MPL3115A2.py pysense2/lib/
+	cp shields/lib/SI7006A20.py pysense2/lib/
+	# pycoproc
+	cp shields/lib/pycoproc.py pysense/lib/pycoproc.py
+	# example
+	cp shields/pysense_2.py pysense/main.py
```
Comment (peter-pycom) [inline Makefile:30]: shields_2.py should go to main.py that is the one that has the sleep/wake stuff so it should run from main.py

**Review by peter-pycom — COMMENTED** [review]
thanks!

File: Makefile (hunk 2 above)
Comment (gijsio) [reply Makefile:25]: I feel like people might want to use the temp example folder as a project, instead of having to manually unzip the folder

**Review by peter-pycom — COMMENTED** [review]
as discussed, let's try to merge the pybytes and non-pybytes versions, otherwise we end up with 10 release artefacts. 

also pls use the same order of things, so, if it's scan, sense, track (ok, alphabetical :) ) followed by sense, track for 2 - fine. then it should be the same order everywhere

File: shields/pyscan_1.py [hunk 5]
```diff
@@ -28,6 +36,12 @@
 lt = LTR329ALS01(py)
 li = LIS2HH12(py)
 
+pybytes_enabled = False
+if 'pybytes' in globals().keys():
```
Comment (peter-pycom) [inline shields/pyscan_1.py:40]: no keys()

**Review by peter-pycom — CHANGES_REQUESTED** [review]

**Pull request merged**